sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
 ## Release History
 
-### 4.15.0-beta.2 (Unreleased)
+### 4.15.0-beta.3 (Unreleased)
```

Comment:
You don't need to add these yet. These are added automatically after the release. Please remove it here and from encryption changelog as well.

Reply:
Thanks, please check now